src/textwin.cpp

```diff
@@ -163,8 +219,19 @@ static ShowHideButton pointsButton =
     { &(SS.GW.showPoints),       "point",         "points"                          };
 static ShowHideButton constructionButton =
     { &(SS.GW.showConstruction), "construction",  "construction entities"           };
-static ShowHideButton constraintsButton =
+
+/*static ShowHideButton constraintsButton =
```

Comment:
Let's not leave old code in comments. That's what git is for, just remove the commented-out part.